StardewArchipelago/Locations/CodeInjections/Modded/SVE/SVECutsceneInjections.cs

```diff
@@ -109,7 +106,7 @@ public static bool EndBehaviors_AddSpecialOrderAfterEvent_Prefix(string[] split,
                 }
                 //Change the key so it doesn't get deleted
                 var eventsKey = sveEventSpecialOrders[__instance.id];
-                if (eventsKey == "Apples") // Changes apples quest with one relevant to bundle price
+                if (eventsKey == APPLES_NAME) // Changes apples quest with one relevant to bundle price
                 {
                     eventsKey = AuroraVineyardMatchesBundles();
```

Comment:
Not a big fan of this method name. How about "GetAuroraVineyardPricedEventKey()", to better convey what happens in it without having to look?